src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -121,7 +121,7 @@ class DoorLockServer
     bool SetOneTouchLocking(chip::EndpointId endpointId, bool isEnabled);
     bool SetPrivacyModeButton(chip::EndpointId endpointId, bool isEnabled);
 
-    bool TrackWrongCodeEntry(chip::EndpointId endpointId);
+    bool HandleWrongCodeEntry(chip::EndpointId endpointId);
```

Comment:
Should there be followup issues to update the example apps to call this?

Reply:
yes agreed. will file one to track.

Reply:
https://github.com/project-chip/connectedhomeip/issues/25599